Jellyfin.Api/Controllers/FilterController.cs

```diff
@@ -112,6 +114,20 @@ public FilterController(ILibraryManager libraryManager, IUserManager userManager
                 .Where(i => !string.IsNullOrWhiteSpace(i))
                 .Distinct(StringComparer.OrdinalIgnoreCase)
                 .Order()
+                .ToArray(),
+
+            AudioLanguages = GetMediaStreams(itemList, BaseItemKind.Movie, BaseItemKind.Episode)
```

Comment:
GetMediaStreams is a database call for at least every item in the itemList. Please put `GetMediaStreams(itemList, BaseItemKind.Movie, BaseItemKind.Episode).ToList()` in a variable as you are calling it twice.

Reply:
fixed